horton/io/iodata.py

```diff
@@ -237,6 +232,12 @@ def __init__(self, **kwargs):
         for key, value in kwargs.iteritems():
             setattr(self, key, value)
 
+    # Numpy array attributes that may require orbital basis reordering or sign correction.
```

Comment:
This is a temporary hack and it seemed too early to work out a nice solution. I'll ad some comments on how this should be done better when factoring the io code.